assemblers/org.osbuild.qemu

```diff
@@ -119,15 +119,15 @@ STAGE_OPTS = """
   "pttype": {
     "description": "The type of the partition table",
     "type": "string",
-    "enum": ["mbr", "gpt"]
+    "enum": ["mbr", "dos", "gpt"]
   },
   "root_fs_uuid": {
     "description": "UUID for the root filesystem",
     "type": "string"
   },
   "size": {
     "description": "Virtual disk size",
```

Comment:
Sorry for commenting on a merged PR :) Should all of the 'size' field descriptions include units? Or are they all assumed to be bytes?

Reply:
They're always bytes.